include/event_service_store.hpp

```diff
@@ -258,6 +258,31 @@ class EventServiceStore
     {
         return eventServiceConfig;
     }
+
+    bool needsWrite()
+    {
+        return needWrite;
+    }
+
+    void persistSubscription()
+    {
+        needWrite = true;
+        return;
+    }
+    void removeSubscription(const std::string& id)
+    {
+        auto obj = subscriptionsConfigMap.find(id);
+        if (obj != subscriptionsConfigMap.end())
+        {
+            BMCWEB_LOG_ERROR << "Deleting " << id
+                             << " from subscriptionsConfigMap";
+            subscriptionsConfigMap.erase(obj);
+            needWrite = true;
+            return;
+        }
+        BMCWEB_LOG_ERROR << "Subscriber " << id
```

Comment:
Is this really needed as an LOG ERROR?

Reply:
This is hit if there is an error actually. ie the subscriber is asked to be removed, but its not found at the subscription config map